src/app/features/spell/spell-dbc/base/spell-dbc-base.component.html

```diff
@@ -2,276 +2,328 @@
+					[tooltip]="'Base mana cost of a spell. It is being summed with ManaCostPercentage.'"
+				></i>
+			<input [formControlName]="'ManaCost'" id="ManaCost" type="number" class="form-control form-control-sm">
+		</div>
+
+		<div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+			<label class="control-label" for="ManaCostPerLevel">ManaCostPerLevel</label>
+				<i
+					class="fas fa-info-circle ml-1" [placement]="'auto'"
+					[tooltip]="'Likely is mana cost increased by ((caster level) - (spell level))*(this field).'"
+				></i>
+			<input [formControlName]="'ManaCostPerLevel'" id="ManaCostPerLevel" type="number" class="form-control form-control-sm">
+		</div>
+
+		<div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+			<label class="control-label" for="ManaPerSecond">ManaPerSecond</label>
+				<i
+					class="fas fa-info-circle ml-1" [placement]="'auto'"
+					[tooltip]="'For channels.'"
+				></i>
+			<input [formControlName]="'ManaPerSecond'" id="ManaPerSecond" type="number" class="form-control form-control-sm">
+		</div>
+
+		<div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+			<label class="control-label" for="ManaPerSecondPerLevel">ManaPerSecondPerLevel</label>
+				<i
+					class="fas fa-info-circle ml-1" [placement]="'auto'"
+					[tooltip]="'For channels. Likely is mana cost per second increased by ((caster level) - (spell level))*(this field).'"
+				></i>
+			<input [formControlName]="'ManaPerSecondPerLevel'" id="ManaPerSecondPerLevel" type="number" class="form-control form-control-sm">
+		</div>
+
+		<div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+			<label class="control-label" for="ManaCostPct">ManaCostPct</label>
+				<i
+					class="fas fa-info-circle ml-1" [placement]="'auto'"
+					[tooltip]="'% of base mana cost, added to flat value in manaCost field. For NPCs is base mana = their max mana.'"
+				></i>
+			<input [formControlName]="'ManaCostPct'" id="ManaCostPct" type="number" class="form-control form-control-sm">
+		</div>
+
+		<div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+			<label class="control-label" for="PreventionType">PreventionType</label>
+				<i
+					class="fas fa-info-circle ml-1" [placement]="'auto'"
+					[tooltip]="'TODO'"
+				></i>
+			<input [formControlName]="'PreventionType'" id="PreventionType" type="number" class="form-control form-control-sm">
+		</div>
+
+		<div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+			<label class="control-label" for="RequiresSpellFocus">RequiresSpellFocus</label>
+				<i
+					class="fas fa-info-circle ml-1" [placement]="'auto'"
+					[tooltip]="'Whether spell focus is needed by spell to be cast. Player needs to be within range of appropriate gameobject with type=8.'"
+				></i>
+			<input [formControlName]="'RequiresSpellFocus'" id="RequiresSpellFocus" type="number" class="form-control form-control-sm">
+		</div>
+	</div>
```

Comment:
it should be 2 spaces instead of a tab, but I can take care of this in another PR...